### PR TITLE
feat(ListView): Add icons support to ListView component

### PR DESCRIPTION
**Pull request by akaseka**
**What is the problem this PR is trying to solve?**
 Add icons support to ListView component

**What is the chosen solution to this problem?**
Adde icon props to ListView component

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/ListView/Items/Item/Item.component.js [hunk 1]
```diff
@@ -66,6 +68,13 @@ class Item extends Component {
 					label={searchCriteria ? getSearchedLabel(item.label) : item.label}
 					aria-label={ariaLabel}
 				/>
+				{item.icon && (
+					<TooltipTrigger label={item.icon.title} tooltipPlacement="bottom">
+						<span>
```
Comment (frassinier) [inline packages/components/src/ListView/Items/Item/Item.component.js:73]: why this extra span?

File: packages/components/src/ListView/Items/Item/Item.component.js [hunk 2]
```diff
@@ -66,6 +68,13 @@ class Item extends Component {
...
+							<Icon {...item.icon} aria-hidden="false" />
```
Comment (frassinier) [inline packages/components/src/ListView/Items/Item/Item.component.js:74]: Icons are aria-hidden by default

File: packages/components/src/ListView/ListView.stories.js [hunk 3]
```diff
@@ -179,6 +197,11 @@ storiesOf('Form/Controls/ListView', module)
 			</div>
 		);
 	})
+	.add('with icon', () => (
```
Comment (frassinier) [inline packages/components/src/ListView/ListView.stories.js:200]:
```suggestion
	.add('with icons', () => (
```

File: packages/components/src/ListView/Items/Item/Item.component.js (hunk 2 above)
Comment (jsomsanith-tlnd) [reply packages/components/src/ListView/Items/Item/Item.component.js:74]: Are the icons only decorative ? For example next to the displayed text.
If so, it should stay hidden you can remove this extra aria-hidden (default in Icon is true as Fabien said).
If not, they must have a text-alternative.

File: packages/components/src/ListView/Items/Items.scss [hunk 4]
```diff
@@ -16,6 +16,13 @@ $row-nested-inner-margin-bottom: $padding-larger - $padding-small;
 		:global {
 			.checkbox {
 				margin-left: $padding-smaller;
+				display: flex;
+			}
+			:global(.tc-svg-icon) {
```
Comment (jsomsanith-tlnd) [inline packages/components/src/ListView/Items/Items.scss:21]: you already are in a `:global`
```suggestion
			.tc-svg-icon {
```

File: packages/components/src/ListView/Items/Item/Item.component.js (hunk 1 above)
Comment (akaseka) [reply packages/components/src/ListView/Items/Item/Item.component.js:73]: @frassinier I took it from Enumeration, without span  the icon becomes a button
image

**Review by frassinier — APPROVED** [review]

File: packages/components/src/ListView/Items/Item/Item.component.js (hunk 1 above)
Comment (jmfrancois) [reply packages/components/src/ListView/Items/Item/Item.component.js:73]: it seems tooltip trigger is adding the onClick

File: packages/components/src/ListView/Items/Item/Item.component.js (hunk 2 above)
Comment (akaseka) [reply packages/components/src/ListView/Items/Item/Item.component.js]: yes icon is decorative with a hint

**Review by jmfrancois — APPROVED** [review]

**Pull request merged**